File: desktop-currency-converter/utils/price-tag-maker.js
// List of extended currency signs
const extendedCurrencies = ['$', '€', '£', '¥', '₩', '₹', '₽', '₱', '฿', 'CHF', 'C$', 'A$', 'NZ$', 'HK$', 'S$'];

// Function to generate a random price with a currency sign from the list
function generateRandomPrice() {
    const amount = (Math.random() * (1000 - 1) + 1).toFixed(2);
    const currency = extendedCurrencies[Math.floor(Math.random() * extendedCurrencies.length)];
    return `${currency}${amount}`;
}

// Generate a list of 10 random price tags
let priceTags = Array.from({ length: 10 }, () => generateRandomPrice());

// Convert priceTags to a string of <p> tags
let priceTagsHTML = priceTags.map(tag => `<p>${tag}</p>`).join('');

// Output the list of price tags
console.log(priceTags);

// Later use: document.div.innerHTML = priceTagsHTML; to display in HTML
document.getElementById('priceContainer').innerHTML = priceTagsHTML;

File: desktop-currency-converter/utils/price_tag_maker.py
import random

# List of extended currency signs
extended_currencies = ['$', '€', '£', '¥', '₩', '₹', '₽', '₱', '฿', 'CHF', 'C$', 'A$', 'NZ$', 'HK$', 'S$']

# Function to generate a random price with a currency sign from the list
def generate_random_price():
    amount = round(random.uniform(1, 1000), 2)
    currency = random.choice(extended_currencies)
    return f"{currency}{amount}"

# Generate a list of 10 random price tags
price_tags = [generate_random_price() for _ in range(10)]

# Output the list of price tags
print(price_tags)
